### PR TITLE
added session variables

### PR DESCRIPTION
**Pull request by aulus-plautius**
Added two session variables:
username
user_id

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import os
-from flask import Flask, request, render_template
+from flask import Flask, request, render_template, session
 from lib.database_connection import get_flask_database_connection
 from routes.user_routes import *
 
 # Create a new Flask app
 app = Flask(__name__)
-
+app.secret_key = "psyducks_x_slowbros"
```
Comment (MyGollyGosh) [inline app.py:8]: Great key!

File: tests/test_user_routes.py [hunk 2]
```diff
@@ -55,6 +55,36 @@ def test_create_account_goes_to_sign_in(web_client, test_web_address, db_connect
...
+    username_status_element = page.locator(".current_user_username")
+    expect(username_status_element).to_have_text("guest1")
+
+def test_user_signs_in_with_incorrect_email_but_correct_password(web_client, test_web_address, db_connection, page: Page):
+    db_connection.seed('seeds/makersbnb.sql')
+    page.goto(f'http://{test_web_address}/sign-in')
+    page.fill("input[name='email']", 'guest1@exmple.com')
+    page.fill("input[name='password']", 'password1')
+    page.locator('.signin').click()
+    page.screenshot(path='screenshot.png', full_page=True)
```
Comment (MyGollyGosh) [inline tests/test_user_routes.py:74]: Can we remove screenshot please

**Review by MichaelVerdon — APPROVED** [review]
Gwan son

File: tests/test_user_routes.py [hunk 3]
```diff
@@ -55,6 +55,36 @@ def test_create_account_goes_to_sign_in(web_client, test_web_address, db_connect
...
+    sign_in_message = page.locator(".sign_in_message")
+    expect(sign_in_message).to_have_text("Invalid email or password")
+
+def test_user_signs_in_with_correct_email_incorrect_password(web_client, test_web_address, db_connection, page: Page):
+    db_connection.seed('seeds/makersbnb.sql')
+    page.goto(f'http://{test_web_address}/sign-in')
+    page.fill("input[name='email']", 'guest1@example.com')
+    page.fill("input[name='password']", 'password')
+    page.locator('.signin').click()
+    page.screenshot(path='screenshot.png', full_page=True)
```
Comment (MyGollyGosh) [inline tests/test_user_routes.py:84]: Can we remove screenshot

File: app.py (hunk 1 above)
Comment (kikilondon) [reply app.py:8]: Well done

**Review by kikilondon — APPROVED** [review]

**Review by Alexia-May — APPROVED** [review]
Good to have sessions!

**Pull request merged**